src/en/help-oracle.md

```diff
@@ -2,94 +2,94 @@ Title: Using the Oracle cloud
-the following:
+1. Navigate to 'My Services URL' and log in.
+1. Select your 'Identity Domain' in the top-right corner.
+1. Click on the 'Create instance' box and then 'Create' a Compute service.
 
-[https://myservices.emea.oraclecloud.com](https://myservices.emea.oraclecloud.com).
+![Create Compute service](./media/oracle_empty-dashboard-2.png)
 
-After signing in, you'll see the default top level of Oracle's domain
-dashboard.
+!!! Recall:
+    We are doing this to associate images with your 'identity domain'. We will
+    not be creating an instance here.
 
-![Empty Oracle dashboard](./media/oracle_empty-dashboard.png)
+Click on 'Marketplace' on the resulting page (left menu), enter 'ubuntu' into
+the search field, and hit Enter:
 
-Click on the large `Create Instance` tile and select `Compute` from the list of
-services that appear. 
+![Search Ubuntu images](./media/oracle_create-instance-ubuntu-2.png) 
 
-You are now looking at the first step in Oracle's Compute cloud deployment
-process. We'll be using this step to associate images with this specific cloud
-domain before cancelling the remainder of Oracle's cloud deployment. 
+From the point of view of Juju, some images should not be used.
+Juju-compatible images are listed below:
 
-Ubuntu images can be found by clicking on 'Marketplace' from the menu on the
-left and entering `Ubuntu` into the search field that appears. 
+| Version          | Arch   | Series  |
+|------------------| -------|---------|
+| Ubuntu 12.04 LTS | amd64  | Precise |
+| Ubuntu 14.04 LTS | amd64  | Trusty  |
+| Ubuntu 16.04 LTS | amd64  | Xenial  |
+| Ubuntu 17.04     | amd64  | Zesty   |
 
-![Ubuntu image search](./media/oracle_create-instance-ubuntu.png) 
+Since Juju uses charms to install applications, the Ubuntu series you need are
+those that the charms were written for. If unsure, it is recommended to add the
+two most recent LTS releases.
 
-The search will return any official Ubuntu images, visible with the Ubuntu
-logo, alongside other Ubuntu-associated images. 
+!!! Note:
+    At time of writing, Trusty and Xenial are the two most recent Ubuntu
+    LTS releases.
 
-It's currently possible to use any of the following images with Juju:
+Select a compatible image from among the official Ubuntu images (orange Ubuntu
+logo), accept Oracle's terms and conditions, and click 'Install'. Repeat the
+process for each desired image. These installed images will end up under
+'Private Images' in the menu on the left:
 
-| Version          | Arch   | Series  |
-|------------------| -------|---------|
-| Ubuntu 12.04-LTS | amd64  | Precise |
-| Ubuntu 14.04-LTS | amd64  | Trusty  |
-| Ubuntu 16.04-LTS | amd64  | Xenial  |
-| Ubuntu 17.04     | amd64  | Zesty   |
+![List private images](./media/oracle_create-instance-private-2.png)
 
-!!! Warning: 
-	Currently, Oracle's Ubuntu 16.10 (Yakkety Yak) image shouldn't be
-        associated with your Juju/Oracle Compute deployment.
+## Add cloud (trial accounts)
```

Comment:
What if we call the header 'Trial accounts' instead of 'Add cloud (trial accounts)'?